src/main.rs

```diff
@@ -146,20 +162,104 @@ fn collect_information_and_write_to_file(out_path: String) {
         if let Some(message) = maybe_message {
             let result = format!("{} {}\n", emoji, message);
 
-            let mut f = File::create(out_path).unwrap();
+            let mut f = File::create(out_path.clone()).unwrap();
             f.write_all(result.as_bytes()).unwrap();
+
+            if launch_editor {
+                launch_default_editor(out_path);
```

Comment:
Maybe you want to `drop(f)` so that the file is always written to disk before calling out to the external editor? 🤔 
```suggestion
            drop(f);

            if launch_editor {
                launch_default_editor(out_path);
```